index.pug

```diff
@@ -221,15 +223,7 @@ html(lang='en')
                     p
                       | Invite your inhouse translators giving them the rights for the languages needed.
                     p
-                      | Got a language not covered? Need extra help or prefer to completely outsource the job? Choose one of our 
```

Comment:
would be nice to have that part back again....

Reply:
the main slogan would be mentioned in h3 but this could not be changed as this is i18next script. 

h3 suggestion was 

Bridging the gap between translation and development with locize, a  modern and affordable localization management platform. It makes your website, app, game or whatever your project is, global, vibrant, and more engaging especially when unleashing the continuous localization capabilities.


h1 suggestion: h1 suggestion: localization management platform for continuous localization
h1 localization management platform - Bridging the gap between translation and development. (not possible to SEO as it's too long max 600px long)

We need main keyword in title otherwise the site would not rank for localization management platform.